### PR TITLE
Test_addExportPolicies

### PR DESCRIPTION
**Pull request by andrewsykim**
* also fixes bug where default policy was temporarily set to `ACCEPT`   

cc @murali-reddy @roffe

### CODE REVIEW

File: app/controllers/network_routes_controller.go [hunk 1]
```diff
@@ -753,18 +753,19 @@ func (nrc *NetworkRoutingController) addExportPolicies() error {
 		err = nrc.bgpServer.AddPolicyAssignment("",
 			table.POLICY_DIRECTION_EXPORT,
 			[]*config.PolicyDefinition{&definition},
-			table.ROUTE_TYPE_ACCEPT)
```
Comment (andrewsykim) [inline app/controllers/network_routes_controller.go]: assumed this was a bug

**Comment by andrewsykim** [comment]
sorry, don't review yet actually, there's some changes I'd like to make :)

**Comment by andrewsykim** [comment]
ready for review now

**Comment by murali-reddy** [comment]
thanks @andrewsykim

So if i understand the changes in `app/controllers/network_routes_controller.go` though in case of `if !policyAssignmentExists {` we were doing `table.ROUTE_TYPE_ACCEPT`, but as it follwed by subsequnet replace with `table.ROUTE_TYPE_REJECT` things worked?

**Comment by andrewsykim** [comment]
That was the impression I had, I can leave it so that `Replace` is always called, to be consistent with other parts of the function.

**Comment by murali-reddy** [comment]
I verfied change in this PR. Export policies are as expected.

- default export policy to reject everything
- explicit accept policy statement to advertise node CIDR and cluster ip's

```
#gobgp global policy export
Export policy:
Default: reject
Name kube_router:
    StatementName kube_router_stmt0:
      Conditions:
        PrefixSet: any podcidrprefixset
      Actions:
         accept
    StatementName kube_router_stmt1:
      Conditions:
        PrefixSet: any clusteripprefixset
        NeighborSet: any externalpeerset
      Actions:
         accept
```

**Pull request merged**